### PR TITLE
Introduce MokoResources for the project

### PR DESCRIPTION
**Pull request by kittinunf**
## Issue
- #259 but it can't close the issue just yet because we have to make sure it works on iOS.

## Overview (Required)
- In this PR, we are introducing the moko-resource to maintain/manage our string files.

Now I have only migrated the about section first. First, if this can move forward, I would move on to the rest of the app.

## Links
-

## Screenshot
In Jp:

Screen Shot 2022-09-12 at 12 39 48

In En:

Screen Shot 2022-09-12 at 12 44 55

### CODE REVIEW

File: core-model/build.gradle.kts [hunk 1]
```diff
@@ -22,4 +23,15 @@ kotlin {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+dependencies {
```
Comment (kittinunf) [inline core-model/build.gradle.kts:28]: @takahirom I know this is questionable but I need a place where we could satisfy the direct exposure to the ios-combined module. Now the options are;

```
val projects = listOf(
            projects.coreModel,
            projects.coreDesignsystem,
            projects.coreData
        )
```

I found `coreModel` to be the sanest one.

File: core-model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/strings/ResExt.kt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package io.github.droidkaigi.confsched2022.strings
+
+import io.github.droidkaigi.confsched2022.model.Res
+import io.github.droidkaigi.confsched2022.model.Res.strings
+
+val Strings: strings
```
Comment (kittinunf) [inline core-model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/strings/ResExt.kt:6]: Find this to be easier than spell it out the whole thing :)

File: core-model/build.gradle.kts (hunk 1 above)
Comment (takahirom) [reply core-model/build.gradle.kts:28]: I don't dislike this idea. 👀 
But originally, I added core resources.
I have a question about this. Can we use string value in the model class?

https://github.com/DroidKaigi/conference-app-2022/pull/119/files#diff-c053860dd51d0a036f0a1ffc0adbca71dd1dcf8f9610e502da3e1c1f02db1f68R12

By the way, can you move it here?

```
kotlin {
    sourceSets {
        val commonMain by getting {
            dependencies {
// here

Comment (kittinunf) [reply core-model/build.gradle.kts:28]: Sure I can! 

I don't see a reason why we can't use the string value in the model class. It should be possible.

**Comment by takahirom** [comment]
I checked it in my local 👍  It's great! Let's merge then. Can you resolve the conflict?

**Comment by kittinunf** [comment]
Yes! I will sorry got swamped with work stuff. I will work on this and resolve it tonight 💦

**Comment by takahirom** [comment]
> I will help refactor to use moko resource in all others screens later.

Thank you!

**Pull request merged**